packages/common/src/Utils.ts

```diff
@@ -253,3 +257,25 @@ export function getDataAndSignature (tx: Transaction, chainId: number): { data:
 export function signedTransactionToHash (signedTransaction: PrefixedHexString, transactionOptions: TxOptions): PrefixedHexString {
   return bufferToHex(Transaction.fromSerializedTx(toBuffer(signedTransaction), transactionOptions).hash())
 }
+
+// remove properties with null (or undefined) value
+// recursive - descend into inner objects
+// (does NOT handle inner arrays)
+export function removeNullValues<T> (obj: T, recursive = false): Partial<T> {
```

Comment:
because otherwise, Ctf can't use
```
const maxViewableGasLimit = chainId === 6 ? 5e6 : undefined
```

since putting `maxViewableGasLimit : undefined`
in the configuration prevent the default value to be filled in...

Reply:
I would actually perform an input verification on the `GSNConfig` and **throw** if the input has invalid types, `undefined`, `null`, `NaN`, `{}`, `false`, `"false"`,  `""` etc. etc.

Why would we silently sanitise the "garbage in"?